package.json

```diff
@@ -60,6 +61,7 @@
     "sass": "^1.42.1",
     "sass-loader": "^10.1.0",
     "style-loader": "^2.0.0",
+    "twig": "^1.15.4",
```

Comment:
Do we have to explicitly require twig for this to work?

Reply:
nope. removed.